src/modules/docs/components/HeaderDocCustom.tsx

```diff
@@ -0,0 +1,48 @@
+import { HTMLAttributes } from 'react'
+import { Anchor, Flex, Text } from 'dd360-ds'
+import { LinkIcon } from '@heroicons/react/solid'
+
+interface TitleProps extends HTMLAttributes<HTMLHeadingElement> {
+  name: string
+}
+
+const titleProps: TitleProps = {
+  name: 'floating-nav'
+}
+
+const HeaderDocCustom = ({ title, url }: { title: string; url?: string }) => {
+  return (
+    <Flex
+      alignItems="center"
+      justifyContent="between"
+      gap="2"
+      className="flex-row flex-wrap w-full"
+    >
+      <h1 {...titleProps}>{title}</h1>
```

Comment:
we can use our component Text just here

Reply:
I can't use the Text component because it complains about typescript when passing the name attribute, and I can't find a way to fix it.